arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -5831,6 +5835,18 @@
         "modelAsString": false
       }
     },
+    "SigningAlgorithm": {
```

Comment:
What I'm trying to say is if you wish to update this list in the future, it would be a breaking change to your sdk. If you'd like to avoid that, consider the comment above